File: teste.rb
require 'websocket-eventmachine-client'

EM.run do
  ws = WebSocket::EventMachine::Client.connect(uri: 'ws://localhost:3000/cable?user_id=1')

  ws.onopen do
    ws.send '{"command":"subscribe","identifier":"{\"channel\":\"ChatChannel\"}"}'
    puts "connected"
  end

  ws.onmessage do |message, type|
    puts "recebeu #{message}, #{type}"
  end

  ws.onclose do |code, reason|
    puts "saiu, code: #{code}, reason: #{reason}"
  end
end

File: teste.js
const WebSocket = require('ws');

function onError(ws, err) {
  console.log(`onError: ${err.message}`);
}

function onMessage(ws, data) {
  console.log(`onMessage: ${data}`);
}

const ws = new WebSocket('ws://localhost:3000/cable?user_id=1')

ws.on('open', function onConnection() {
  console.log('open');

  const identifier = '{"channel":"ChatChannel"}';
  ws.send(JSON.stringify({"command":"subscribe","identifier":identifier}));

  ws.on('message', data => onMessage(ws, data));
  ws.on('error', error => onError(ws, error));
})

console.log('rodando...');
